azurerm/internal/services/network/resource_arm_web_application_firewall_policy.go

```diff
@@ -402,12 +577,102 @@ func flattenArmWebApplicationFirewallPolicyPolicySettings(input *network.PolicyS
 
 	result := make(map[string]interface{})
 
-	result["enabled"] = input.State == network.WebApplicationFirewallEnabledStateDisabled
+	result["enabled"] = input.State == network.WebApplicationFirewallEnabledStateEnabled
```

Comment:
This one snuck in, and fixes a bug where `policy_settings.enabled` is set to the opposite of the actual value.